TotalMayhemGames/EscapeRoom
# Feat/puzzle theater

How to test:

- Start playing the theater puzzle -> walk up to the puppet.
- you should see a vignette effect increase the closer you get to the puppet.
- When standing to close -> you die.
- Set one of the trapdoors in the inspector to unlocked and use it -> screen should fade out and in
